deno/src/utils/BroadcastMethods.ts

```
@@ -40,7 +40,9 @@ export class BroadcastMethods<T extends Record<string, Function>> {
         const result = await methods[fn](...args)
         chan.postMessage({ result, requestId })
       } catch (error) {
-        chan.postMessage({ error, requestId })
+        try{
+            chan.postMessage({ error, requestId })
+        }catch(e){}
```

Is there anyway to match the error for this specific situation as if it's only a development issue the server shouldn't silently fail in production. Also definitely include a `console.error('BroadcastChannel is dead failed to send errror', error)`

---

Honestly i don't think it's dev only...i think the server silently restart in prod so it goes unnoticed. If you have a way to check your production logs we might be able to found out if i'm right. I'll add a console.error btw.